recipes/Cgl/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+UNAME="$(uname)"
+export CFLAGS="${CFLAGS} -O3"
+export CXXFLAGS="${CXXFLAGS} -O3"
+export CXXFLAGS="${CXXFLAGS//-std=c++17/-std=c++11}"
+
+if [ "${UNAME}" == "Linux" ]; then
+    export FLIBS="-static-libgcc -Wl,-Bstatic,-lstdc++,-Bdynamic -lm"
```

Comment:
Why?

Reply:
```suggestion
    export FLIBS="-lgcc -lstdc++ -lm"
```

Reply:
cc @scopatz I copied this from you

Reply:
I guess there shouldn't be any need to set `FLIBS`, as we are not building any Fortran code. But in any case, `configure` should be setting that variable properly, over-riding it should probably be avoided unless there is a reason. 

We don't use any C++11 or C++17. The substitution shouldn't hurt anything, but to keep things simple, it shouldn't be needed.

Reply:
there were linker errors when using C++17. no idea where they come from exactly but I spent too much time to debug them so this is just to be on the safe side.

Reply:
Locally I also had an error when not setting FLIBS. I can try it again when I also remove the fortran compiler.

Reply:
Yes, the configure script requires a fortran compiler even though the package doesn't. So, you have to set FLIBS to override `-lgfortran` in there, but you should remove static linking to libgcc and libstdc++.

Reply:
Hmm, OK. The check for the Fortran compiler is also fixed in the development version, but in these releases, there is still a check for it. This never caused a problem for me, though, even when the Fortran compiler is missing. I'll try to carve out some time to do a local build and see what is happening.